crates/proof-of-sql/src/sql/proof_exprs/range_check_test_expr.rs

```diff
@@ -0,0 +1,125 @@
+use super::range_check::{
+    count_range_check, final_round_evaluate_range_check, verifier_evaluate_range_check,
+};
+use crate::{
+    base::{
+        database::{ColumnField, ColumnRef, OwnedTable, Table, TableEvaluation, TableRef},
+        map::{indexset, IndexMap, IndexSet},
+        proof::ProofError,
+        scalar::Scalar,
+    },
+    sql::proof::{
+        CountBuilder, FinalRoundBuilder, FirstRoundBuilder, ProofPlan, ProverEvaluate,
+        VerificationBuilder,
+    },
+};
+use bumpalo::Bump;
+use serde::Serialize;
+
+#[derive(Debug, Serialize)]
+pub struct RangeCheckTestExpr {
```

Comment:
Since it is actually a `ProofPlan` you want to name it something like `RangeCheckTestPlan`